protocols/grpc/agent.proto

```diff
@@ -170,24 +174,17 @@ message Route {
 	string device = 3;
 }
 
-enum UpdateType {
-	None = 0;
-	AddIP = 1;
-	RemoveIP = 2;
-	Name = 4;
-	MTU = 8;
-}
```

Comment:
If remove this, how to distinguish AddIP and RemoveIP ?

Reply:
The idea is for the runtime to give a complete state of the interface and ask the agent to put the interface in that state. So there would not be any update operation anymore, it would only be: Update the interface to that state (including IP, mac, MTU, etc...). That really is a declarative interface: You only describe the state you want to reach, not the steps in between.

Reply:
I get the point, this sounds so good